templates/node.js-express-webapp-to-azure-on-linux.yml

```diff
@@ -1,5 +1,5 @@
-# Node.js Express Web App on Linux
-# Build a Node.js Express app and deploy it as a Linux web app.
+# Node.js Express Web App to Azure on Linux

```

Comment:
to Linux on Azure

Please do it everywhere

Reply:
Done